src/hdcontroller.ts

```diff
@@ -167,7 +176,7 @@ export class HDProxydClient {
      * This creates a default request for hdproxyd
      */
     private _createRequestHeader(method: string, headers: {[key: string]: string}|undefined,
-                                 key: string, params: any, log: any): object {
+        key: string, params: Params, log: werelogs.RequestLogger): object {
```

Comment:
why reformat this? is this required by eslint?

Reply:
same as here : https://github.com/scality/hdclient/pull/833/files#r1795438326